### ISSUE TITLE
openshift-install 4.2 failing on Azure

### ISSUE THREAD

**Issue by joaotomazio** [openshift/installer#2400]
# Version
```console
$ openshift-install version
./openshift-install v4.2.0
built from commit f96afb99f1ce4f8976ce62f7df44acb24d2062d6
release image quay.io/openshift-release-dev/ocp-release-nightly@sha256:b3ba58c53a3f5e98f53dff425e7e4c87b60f5d49d66213853b79f00f7a8a9448
```

# Platform:
Platform: Azure
Installer: openshift-install-linux-4.2.0-0.nightly, running on Ubuntu 18.04

# What happened?
The installer times out when initializing cluster, on the "Waiting up to 30m0s for the cluster" phase.
```console
INFO Waiting up to 30m0s for the cluster at https://api.openshift4.oc-demo.ml:6443 to initialize... 
DEBUG Still waiting for the cluster to initialize: Working towards 4.2.0-0.nightly-2019-09-23-154647: 99% complete 
DEBUG Still waiting for the cluster to initialize: Working towards 4.2.0-0.nightly-2019-09-23-154647: 99% complete, waiting on authentication, console, image-registry, ingress, monitoring 
DEBUG Still waiting for the cluster to initialize: Working towards 4.2.0-0.nightly-2019-09-23-154647: 99% complete 
DEBUG Still waiting for the cluster to initialize: Some cluster operators are still updating: authentication, console, image-registry, ingress, monitoring 
DEBUG Still waiting for the cluster to initialize: Working towards 4.2.0-0.nightly-2019-09-23-154647: 99% complete 
DEBUG Still waiting for the cluster to initialize: Some cluster operators are still updating: authentication, console, image-registry, ingress, monitoring 
DEBUG Still waiting for the cluster to initialize: Working towards 4.2.0-0.nightly-2019-09-23-154647: 99% complete 
DEBUG Still waiting for the cluster to initialize: Some cluster operators are still updating: authentication, console, image-registry, ingress, monitoring 
DEBUG Still waiting for the cluster to initialize: Working towards 4.2.0-0.nightly-2019-09-23-154647: 99% complete 
DEBUG Still waiting for the cluster to initialize: Some cluster operators are still updating: authentication, console, image-registry, ingress, monitoring 
FATAL failed to initialize the cluster: Some cluster operators are still updating: authentication, console, image-registry, ingress, monitoring
```
It looks like initializations/updates of some operators are on endless loops, timing out the process. While the installation was stalled, the following output was produced on a parallel terminal:

```console
$ oc --config ./OCP4/auth/kubeconfig get nodes
NAME                        STATUS   ROLES    AGE   VERSION
openshift4-6rxh9-master-0   Ready    master   16m   v1.14.6+c4799753c
openshift4-6rxh9-master-1   Ready    master   16m   v1.14.6+c4799753c
openshift4-6rxh9-master-2   Ready    master   16m   v1.14.6+c4799753c
```

```console
$ oc --config ./OCP4/auth/kubeconfig get co
NAME                                       VERSION                             AVAILABLE   PROGRESSING   DEGRADED   SINCE
authentication                                                                 Unknown     Unknown       True       15m
cloud-credential                           4.2.0-0.nightly-2019-09-23-154647   True        True          True       18m
cluster-autoscaler                         4.2.0-0.nightly-2019-09-23-154647   True        False         False      12m
console                                    4.2.0-0.nightly-2019-09-23-154647   Unknown     True          False      14m
dns                                        4.2.0-0.nightly-2019-09-23-154647   True        False         False      18m
image-registry                                                                 False       True          False      14m
insights                                   4.2.0-0.nightly-2019-09-23-154647   True        False         False      18m
kube-apiserver                             4.2.0-0.nightly-2019-09-23-154647   True        False         False      17m
kube-controller-manager                    4.2.0-0.nightly-2019-09-23-154647   True        False         False      15m
kube-scheduler                             4.2.0-0.nightly-2019-09-23-154647   True        False         False      16m
machine-api                                4.2.0-0.nightly-2019-09-23-154647   True        False         False      18m
machine-config                             4.2.0-0.nightly-2019-09-23-154647   True        False         False      17m
marketplace                                4.2.0-0.nightly-2019-09-23-154647   True        False         False      13m
monitoring                                                                     False       True          True       8m55s
network                                    4.2.0-0.nightly-2019-09-23-154647   True        False         False      17m
node-tuning                                4.2.0-0.nightly-2019-09-23-154647   True        False         False      15m
openshift-apiserver                        4.2.0-0.nightly-2019-09-23-154647   True        False         False      14m
openshift-controller-manager               4.2.0-0.nightly-2019-09-23-154647   True        False         False      16m
openshift-samples                          4.2.0-0.nightly-2019-09-23-154647   True        False         False      11m
operator-lifecycle-manager                 4.2.0-0.nightly-2019-09-23-154647   True        False         False      17m
operator-lifecycle-manager-catalog         4.2.0-0.nightly-2019-09-23-154647   True        False         False      17m
operator-lifecycle-manager-packageserver   4.2.0-0.nightly-2019-09-23-154647   True        False         False      16m
service-ca                                 4.2.0-0.nightly-2019-09-23-154647   True        False         False      18m
service-catalog-apiserver                  4.2.0-0.nightly-2019-09-23-154647   True        False         False      15m
service-catalog-controller-manager         4.2.0-0.nightly-2019-09-23-154647   True        False         False      15m
storage                                    4.2.0-0.nightly-2019-09-23-154647   True        False         False      13m
```
# What you expected to happen?
It was expected the installer to finish successfully.

# How to reproduce it (as minimally and precisely as possible)?
```console
./openshift-install create cluster --dir OCP4 --log-level debug
```

# Anything else we need to know?
This installation was attempted on several Azure Regions, with every one timming out on the "Waiting up to 30m0s for the cluster" phase.

During the installation, the Master Nodes were successfuly instanced and the Bootstrap Node was destroyed, but the Worker Nodes never appeared on the Resource Group (checking on Azure Portal).

**Comment by abhinavdahiya** [comment]
This is duplicate of https://github.com/openshift/installer/issues/2334

/close
